src/frontend/apps/desk/src/features/members/components/MemberGrid.tsx

```diff
@@ -24,6 +24,18 @@ interface MemberGridProps {
   currentRole: Role;
 }
 
+// FIXME : ask Cunningham to export this type
+type SortModelItem = {
+  field: string;
+  sort: 'asc' | 'desc' | null;
+};
+
+function formatSortModel(sortModel: SortModelItem) {
+  const { field, sort } = sortModel;
+  const attribute = field.includes('user') ? field.split('.')[1] : field;
```

Comment:
If you feel I need to make this function extensible, I can pass a second argument, which would a dict, mapping the current fields to their targeted names. It would work as an adapter